acct/chsh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
You'll want to update this once #87 is merged since then that'll need a python3 shebang too.